force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -1030,6 +1030,30 @@
         <shortDescription>Confirmation message when an auto-enrollment mapping is created</shortDescription>
         <value>{0} auto-enrollment mapping was created.</value>
     </labels>
+    <labels>
```

Comment:
Just noticed that the custom labels aren't matching the master table Julie has.

Reply:
Fixed, had to modify html templates a little to bind format variables